install.sh

```diff
@@ -65,8 +65,26 @@ download_mkectl() {
   then
     arch=x86_64
   fi
-  curl --silent -L -s https://s3.us-east-2.amazonaws.com/packages-stage-mirantis.com/${MKECTL_VERSION}/mkectl_${uname}_${arch}.tar.gz | tar -xvzf - -C $installPath
-  echo "mkectl is now executable in $installPath"
+
+  # Check if the version exists on the server by checking HTTP status code
+  echo "Checking if the specified version exists..."
+  HTTP_STATUS=$(curl -s -o /tmp/mkectl.tar.gz -w "%{http_code}" "https://s3.us-east-2.amazonaws.com/packages-stage-mirantis.com/${MKECTL_VERSION}/mkectl_${uname}_${arch}.tar.gz")
```

Comment:
Now that we have this repo, can't we eliminate the s3 bucket and always pull mkectl from the latest release on this repo?

Reply:
Basically what the BOP install.sh script is doing https://github.com/nwneisen/blueprint/blob/main/scripts/install.sh

Reply:
yes, that's the eventual plan. I am still keeping this as currently, we don't have any mkectl releases here. The installation will start failing otherwise.

Reply:
Everything is still setup to use the docs repo so you've got a free and clean slate here. By the time this `install.sh` script is being used, there will be releases in this repo. I would set it up for the final goal and be done with it.

Reply:
@nwneisen This ticket is targeted for 4.0.0. I might have to update the docs ticket before the next release.

Reply:
Once the doc ticket is done, we should be able to make this quick change(no dependency on docs anymore) once the releases are published here.